automated install/basic-install.sh

```diff
@@ -2625,6 +2647,8 @@ main() {
     # Copy the temp log file into final log location for storage
     copy_to_install_log
 
+    echo 'pasword generation'
```

Comment:
Why do you need this?

Also L 2665, L 2722, L2731, L2738?

Reply:
Well, these are not really necessary, but helped understanding the program flow, resp, where it got stuck.
Do you think these should be removed?

Reply:
> but helped understanding the program flow

I understand you used this to "debug" each step while you where learning each code step, but i think these messages are not needed on the installer.

Please remove them.